src/fiber/network.rs

```diff
@@ -2673,30 +2682,72 @@ where
                         },
                     ..
                 }) => {
-                    info!("Funding transaction {:?} confirmed", &tx_hash);
-                    NetworkActorEvent::FundingTransactionConfirmed(
-                        outpoint,
-                        block_number.into(),
-                        DUMMY_FUNDING_TX_INDEX,
-                    )
+                    tokio::spawn( async move  {
+                        const MAX_GET_BLOCK_TIMESTAMP_RETRY: u8 = 5;
+                        for _ in 0..MAX_GET_BLOCK_TIMESTAMP_RETRY {
+                            match call!(
+                                chain,
+                                |reply| CkbChainMessage::GetBlockTimestamp(
+                                    GetBlockTimestampRequest::from_block_number(block_number.into()), reply
```

Comment:
TxStatus has a `block_hash` field, it's better to use the hash to fetch timestamp.

Reply:
Do we also need to remove all fields with type block_number?

Reply:
I think so